LayoutTests/fast/lists/list-type-translucent-color.html

```diff
@@ -0,0 +1,6 @@
+<!DOCTYPE html>
+<body>
+<li style="list-style-type:square; color:rgba(0,0,0,0.2);"></li>
+<li style="list-style-type:disc; color:rgba(0,0,0,0.2);"></li>
+<li style="list-style-type:circle; color:rgba(0,0,0,0.2);"></li>
```

Comment:
can you wrap this inside an `<ul>`? These list items don't render at all in other browsers otherwise, and also padding/margin will probably differ with the reference.

Reply:
Done.